app/views/two_factor_authentication/options/index.html.slim

```diff
@@ -25,7 +25,9 @@ p.mt-tiny.mb3 = @presenter.info
   = f.button :submit, t('forms.buttons.continue')
 
 br
-- if @presenter.should_display_account_reset_or_cancel_link?
+- if @presenter.ial2? && !FeatureManagement.disallow_ial2_recovery?
```

Comment:
I think we need another conditional here to confirm that docauth is enabled before exposing this feature.

Reply:
Revised.  That docauth enabled conditional is now part of should_display_account_reset_or_cancel_link?